### PR TITLE
done a task e integrando com point view do Ivo. Aprova nois

### PR DESCRIPTION
**Pull request by pietro1704**

### CODE REVIEW

**Review by igdutra — APPROVED** [review]
Monstro demais

File: challengeFinalComponents/HUDView/HUDView.swift [hunk 1]
```diff
@@ -8,22 +8,22 @@
...
+        self.addSubview(pause)
         return pause
     }()
     
-    private lazy var decisionPointsView: DecisionPointsView = {
-        let decision = DecisionPointsView()
-        decision.translatesAutoresizingMaskIntoConstraints = false
+    private lazy var decisionPointsView: PointsView = {
+        let decision = PointsView()
+        self.addSubview(decision)
```
Comment (guilhermeluisdomingues) [inline challengeFinalComponents/HUDView/HUDView.swift:24]: isn't it missing decision.translatesAutoresizingMaskIntoConstraints = false  ?

**Review by guilhermeluisdomingues — COMMENTED** [review]
The testHUDView snapshot is correct? The pause button should be on right side, according to Figma

**Review by guilhermeShima — APPROVED** [review]
nice nice :)

File: challengeFinalComponents/HUDView/HUDView.swift (hunk 1 above)
Comment (pietro1704) [reply challengeFinalComponents/HUDView/HUDView.swift:24]: ta dentro da pointsView

**Pull request merged**